cbmc/proofs/ProcessDHCPReplies/Makefile

```diff
@@ -0,0 +1,77 @@
+################################################################
+# ProcessDHCPReplies
+#
+# The proof depends on one parameters
+#   BUFFER_SIZE is the size of the buffer being parsed
+#
+# We need to know the size of the buffers (the buffer size cannot be
+# unconstrained) because the number of loop iterations that must be
+# unwound depends on it.
+# In the code, useful value is BUFFER_SIZE > 241 = sizeof(DHCHMessage_t))
+#
+
+ENTRY=ProcessDHCPReplies
+
+# This is the network buffer size.
+# Reasonable values are size > 241 = sizeof(DHCPMessage_t))
+# Size smaller than this causes CBMC to fail in simplify_byte_extract
+BUFFER_SIZE = 300
```

Comment:
In the interest of paranoia, shouldn't we instead assume the smallest buffer size that the code can support? Embedded developers are a likely group to skew that direction.